### ISSUE TITLE
Cannot reproduce 75% success rate on LIBERO-10 (both success rate and reward are 0)

### ISSUE THREAD

**Issue by wu-mai** [utiasDSL/clare#8]
Thank you for your excellent work on CLARE! I am trying to reproduce the reported ~75% success rate on LIBERO-10 but encountering issues.

Setup:

Pretrained model: continuallearning/dit_flow_mt_libero_90_pretrain
Script: bash bash/dit_dec.sh (with policy.path updated)
Benchmark: LIBERO-10
Problem:
After training, both eval/pc_success and eval/avg_sum_reward_Libero remain at 0. I also noticed the previous issue (#XX) about the success detection bug, but in my case the reward is also 0, suggesting something more fundamental might be off.

Questions:

Could you confirm the exact steps and hyperparameters to reproduce the LIBERO-10 results?
Is there an updated evaluation script that fixes the success detection bug?
Are there specific environment versions (MuJocoCo, LIBERO, etc.) required?
Any guidance would be greatly appreciated!

**Comment by Zzt-gzdr** [comment]
I resolved this issue through the following two steps:

1. Rendering issue: Due to library version mismatches, both collision models and visual models were being rendered. This caused the model to receive incorrect visual inputs (e.g., blue robotic arms, green tabletops). You can fix this by updating your dependencies to these compatible versions:
```numpy==1.26.4
mujoco==2.3.7
opencv-python==4.10.0.84
opencv-python-headless==4.10.0.84
robosuite==1.4.0
rerun-sdk==0.22.0
```
This step will fixed the reward issue.

2. Code issue: You can update `lerobot_lsy/src/lerobot/scripts/eval_peft.py (around line 224)` by replacing:
```if "final_info" in info:
    successes = [
        info["is_success"] if info is not None else False
        for info in info["final_info"]
    ]
else:
    successes = [False] * env.num_envs
```
with:
```
if "is_success" in info:
    successes = np.asarray(info["is_success"], dtype=bool)

    if "_is_success" in info:
        successes &= np.asarray(info["_is_success"], dtype=bool)

    successes = successes.tolist()

elif "final_info" in info:
    successes = [
        final_info["is_success"] if final_info is not None else False
        for final_info in info["final_info"]
    ]

else:
    successes = [False] * env.num_envs
```
This step will fixed the success rate issue.

Hope it will help you.
